group_vars/all/netbox_sites.yaml

```diff
@@ -0,0 +1,7 @@
+netbox_sites:
+  - name: MGHPCC
+  - name: BU PRB
+  - name: OCT
+    description: Open Cloud Testbed (UMass)
+  - name: MOC
+    description: Mass Open Cloud cages
```

Comment:
you can remove OCT and MOC from this list. I thought those could be used as site groups but then I decided that it's unnecessary. 
We only need two sites: MGHPCC and BU PRB.